apps/drec-ui/tsconfig.json

```diff
@@ -18,7 +18,7 @@
         "resolveJsonModule": true,
         "importHelpers": true,
         "isolatedModules": true,
-        "strictNullChecks": false,
+        "strictNullChecks": true,
```

Comment:
Why did you change this rule?

Reply:
To see any issue before pushing.

Reply:
we have it enable for the API

Reply:
But this can break the build now. You also have to adjust all initial `null`'s throughout the UI

Reply:
I`ll change it back